docs/design/chart_format.md

```diff
@@ -90,16 +87,16 @@ A Chart is composed of the following pieces:
 4. An optional `LICENSE` file
 5. An optional `docs/` directory
 6. An optional `hooks/` directory
```

Comment:
Given that we don't currently implement any `hooks` functionality, I'd rather we drop this directory for now, and then add it back if and when we actually need it.